jeeves/report.py

```diff
@@ -14,8 +14,10 @@
 
 def run_report(config, blockers, preamble_file, template_file, no_email, test_email, server, header):
 
+	# get  supported versions from config or use default one
+	supported_versions = config.get('supported_versions', ['13', '16.1', '16.2'])
 	# fetch all relevant jobs
-	jobs = get_jenkins_jobs(server, config['job_search_fields'])
+	jobs = get_jenkins_jobs(server, config['job_search_fields'], supported_versions)
```

Comment:
And what about reminders?

Reply:
This function is not used in reminder. If a job has owner assigned, it lands in reminder mail. There is no restriction for OSP version in reminder mode.